extensions/vscode/src/commands/install-cli.ts

```diff
@@ -0,0 +1,48 @@
+const cp = require("child_process");
+
+import { window, ProgressOptions } from "vscode";
+
+/**
+ * Installs Dart Frog CLI in the user's system if not already installed.
+ */
+export const installCLI = async (): Promise<void> => {
+  if (!hasDartFrogCliInstalled()) {
+    const options: ProgressOptions = {
+      location: 15,
+      title: "Installing Dart Frog CLI...",
+    };
+    window.withProgress(options, installDartFrogCliVersion);
+  }
+};
+
+/**
+ * Whether the user has Dart Frog CLI installed in their system.
+ *
+ * @returns {boolean} True if the user has Dart Frog CLI installed in their
+ * system, false otherwise.
+ */
+function hasDartFrogCliInstalled(): boolean {
```

Comment:
```suggestion
function isDartFrogCliInstalled(): boolean {
```

It is checking if it is installed, so semantics wise this reads better 😉

Reply:
I agree 💙 ! Since I plan to move this method  for reusability within my next Pull Request I will rename it when I do so.